proseco/fid.py

```diff
@@ -94,6 +97,7 @@ class FidTable(ACACatalogTable):
         "detector",
         "sim_offset",
         "focus_offset",
+        "t_ccd_acq",
```

Comment:
I don't think it is controversial to just require the acquisition temperature too if that is the best temperature to use for the fid offsets.